SU-SOE/stanford_soe_helper
# SOE-3211: Adding feature change and css

# READY FOR REVIEW


# Summary
See https://stanfordits.atlassian.net/browse/SOE-3211
- This changes the authoring experience for the top banner (formerly the featured image) on the Stanford Page.
- This add the CTA button to support the "Make a Gift" button. See:
https://stanfordits.atlassian.net/browse/SOE-3158

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this for Giving pages


# Steps to Test
- Switch to this branch: `7.x-2.x-soe-3211-top-banner-styling`
- Clear cache so Drupal can find the module: `drush cc all`
- Revert feature: `drush fr stanford_soe_helper_page`
- Create a new Stanford Page at: `node/add/stanford-page`
- Open the **Add/Edit Top Banner**
- Verify groupings as defined in [SOE-3211](https://stanfordits.atlassian.net/browse/SOE-3211)
NOTE: If **body** and **Call to Action button** are reversed, go to `admin/structure/types/manage/stanford_page/form`, edit (move the **Call to Action** button around), and save.

Also: Because the **Accent Color** is a special case and shared, please set the **Default Value** to **turquoise** at:
`admin/structure/types/manage/stanford-page/fields/field_s_top_banner_color`

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
QA: https://stanfordits.atlassian.net/browse/SOE-3212
Work: https://stanfordits.atlassian.net/browse/SOE-3211 
Styling the CTA: https://stanfordits.atlassian.net/browse/SOE-3158

## Related PRs

## More Information
The styling ticket https://stanfordits.atlassian.net/browse/SOE-3158 will depend on this PR.

## Folks to notify
@boznik 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
